SpatialGDK/Documentation/content/glossary.md

```diff
@@ -30,6 +28,15 @@ An Unreal Editor toolbar command which takes a set of Unreal classes and generat
 ### Singleton Actor
 A server-side authoritative Actor that is restricted to one instantiation on SpatialOS. See documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
 
+### <ProjectRoot>
```

Comment:
This is weird... when I click **View** to see what these new entries look like in context, the headings disappear: 

![image](https://user-images.githubusercontent.com/32169118/47444733-0f418180-d7af-11e8-9723-d9dc842a57e7.png)

Could be to do with the < > ? Any ideas?

Reply:
Yeah that is the html tagging kicking in, will update